2-mandatory/4-stocks.js

```diff
@@ -65,6 +90,17 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
+    let arr = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+      arr.push(
```

Comment:
It's more readable to break this line up; first, work out the variable values. Then, assemble the string.